test/test_build.py

```diff
@@ -663,17 +657,52 @@ def test_multi_build_request(images):
     assert artifacts == expected
 
 
-def assert_fs_customizations(image_type, mountpoint_sizes):
+def assert_fs_customizations(image_type, test_vm):
     """
     Asserts that each mountpoint that appears in the build configuration also appears in mountpoint_sizes.
 
     TODO: assert that the size of each filesystem (or partition) also matches the expected size based on the
     customization.
     """
-    fs_customizations = testutil.create_filesystem_customizations(image_type.rootfs)
-    for fs in fs_customizations:
+    # check the minsize specified in the build configuration for each mountpoint against the sizes in the image
+    # TODO: replace 'df' call with 'parted --json' and find the partition size for each mountpoint
+    exit_status, output = test_vm.run("df --output=target,size", user="root",
+                                      keyfile=image_type.ssh_keyfile_private_path)
+    assert exit_status == 0
+    # parse the output of 'df' to a mountpoint -> size dict for convenience
+    mountpoint_sizes = {}
+    for line in output.splitlines()[1:]:
+        fields = line.split()
+        # Note that df output is in 1k blocks, not bytes
+        mountpoint_sizes[fields[0]] = int(fields[1]) * 2 ** 10  # in bytes
+
+    cfg = {
+        "customizations": {},
+    }
+    testutil.maybe_create_filesystem_customizations(cfg, image_type)
+    for fs in cfg["customizations"]["filesystem"]:
         mountpoint = fs["mountpoint"]
         if mountpoint == "/":
             # / is actually /sysroot
             mountpoint = "/sysroot"
         assert mountpoint in mountpoint_sizes
+
+
+def assert_disk_customizations(image_type, test_vm):
+    exit_status, output = test_vm.run("findmnt --json", user="root",
+                                      keyfile=image_type.ssh_keyfile_private_path)
+    assert exit_status == 0
+    findmnt = json.loads(output)
+    if dc := image_type.disk_config:
+        if dc == "lvm":
+            mnts = [mnt for mnt in findmnt["filesystems"][0]["children"]
+                    if mnt["target"] == "/sysroot"]
+            assert len(mnts) == 1
+            assert "/dev/mapper/vg00-rootlv" == mnts[0]["source"]
```

Comment:
Ideally we would use the config to create `vg00-rootlv` dynamically from the input, but this is ok.